tests/security/test_tls.py

```diff
@@ -0,0 +1,44 @@
+import pytest
+import subprocess
+
+# This test is designed to verify that TLS connections is secured
+#
+# Requirements:
+# - An XCP-ng host
+
+@pytest.mark.parametrize("protocol", ["tls1", "tls1_1"])
+def test_tls_disabled(host, protocol):
+    """
+    Verifies that specified TLS protocols are disabled on the XCP-ng host.
+    Uses the openssl command via subprocess. Should raise CalledProcessError.
+    """
+    PORT = 443
+
+    with pytest.raises(subprocess.CalledProcessError) as execInfo:
+        subprocess.run(
+            ["openssl", "s_client", "-connect", f"{host}:{PORT}", f"-{protocol}"],
+            capture_output=True,
+            text=True,
+            check=True
+        )
+    assert (
+        "no protocols available" in execInfo.value.stderr
+    ), f"{protocol} should be disabled, but got: {execInfo.value.stderr}"
```

Comment:
Are we OK with ignoring stdout here?  Is stderr giving sufficient info when such a protocol is accepted?